recipes/dataprep/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "dataprep" %}
+{% set version = "0.2.10a0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: dd74c71bbeb9faf10249dc8d20a5fe0f16551bf555c7ddcf593f8118141d1421
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  build:
+    - pip
+    - python >=3.6
+    - poetry >=1
+  run:
+    - python >=3.6
+    - bokeh >=2.0,<2.1
+    - dask[complete] >=2.13,<2.14
```

Comment:
conda does not recognize features like this. Please list all dependencies separately

Reply:
@scopatz fixed. BTW do you know if there's any tool I can directly convert pypi requirements to conda format? What I did is basically copy/past/dedupe

Reply:
Nope, there isn't one yet